src/backend/distributed/utils/maintenanced.c

```diff
@@ -356,6 +361,29 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 		}
 #endif
 
+		if (TimestampDifferenceExceeds(lastMetadataSyncTime,
+									   GetCurrentTimestamp(),
+									   METADATA_SYNC_TIMEOUT) && false)
```

Comment:
`&& false` should be removed

Reply:
Currently `SyncMetadataToNodes()` creates a deadlock when run concurrently with `start_metadata_sync_to_node()`, so I've disabled this block temporarily so I can come back to it after fixing other problems.